tests/tests/test-gas.ts

```diff
@@ -80,4 +80,47 @@ describeWithMoonbeam("Moonbeam RPC (Gas)", `simple-specs.json`, (context) => {
 
     expect(await contract.methods.multiply(3).estimateGas()).to.equal(21204);
   });
+
+  it("gas limit should be fine under the weight limit", async function () {
+    const maxBlockTxGas = 2000000 * 0.65; // 2M per block limited to 65% for transactions
```

Comment:
Do we have a constant for gas per block? Or perhaps explain the relationship with these numbers to GAS_PER_SEC, etc.

Reply:
I usually try to avoid relying on external constant in the tests, specially on the integration ones. 
The reason is to make sure when those value changes, the meaning of the test is well understood and manually fixed by the change author.

Right now, our users are going to expect to have 2M gas block limit. If we were relying on a constant and we change that constant to support our new numbers, our test will automatically pass. By forcing them to fail in that case, it makes it clear that we are breaking the expectation of the tools/users using this functionality.

I don't know if I expressed myself clearly but I hope you got the idea :)